src/main/java/com/rakovets/course/java/core/practice/oop/principles/Hero.java

```diff
@@ -0,0 +1,36 @@
+package com.rakovets.course.java.core.practice.oop.principles;
+
+public abstract class Hero implements Mortal{
+    private static String name;
+    private static int health;
```

Comment:
BIG ERROR!
Don't use static for it because static field is ONE for MANY objects.